qt/scientific_interfaces/Indirect/ISISEnergyTransfer.cpp

```diff
@@ -48,6 +50,17 @@ std::string createDetectorGroupingString(std::size_t numberOfDetectors,
                                       numberOfDetectors);
 }
 
+std::vector<std::size_t>
+getCustomGroupingNumbers(std::string const &customString) {
+  std::vector<std::string> customGroupingStrings;
+  std::vector<std::size_t> customGroupingNumbers;
+  // Get the numbers from customString and store them in customGroupingStrings
+  boost::split(customGroupingStrings, customString, boost::is_any_of(" ,-+:"));
+  for (const auto &string : customGroupingStrings)
+    if (string != "")
```

Comment:
`!string.empty()` is preferable for this check.

Reply:
That is better, thanks!